### PR TITLE
Create node manually

### LINKED ISSUE #573: Guide about deploying a node without symbol-bootstrap
Create a guide explaining how to run a node without using `symbol-bootstrap`. The information is mostly in `catapult-server`'s docs.
Reference this guide in the supernode guide (#550) too.

Info from [Slack](https://nemgroup-workspace.slack.com/archives/C015AEQ9ZL2/p1614329284095100):
- peer node - this is peer only node, no api, no mongo (my favorite :heart:)
- api node - ofc requires mongo running and broker, in this case server is passing data to broker, which - well acts as a broker and passes it to both mongo and zmq
- dual node - 2 above in one

---

- mongo - you don't want any repo, you want to install it from packages or use proper docker image - https://docs.mongodb.com/manual/administration/install-community/ - the only options that are interesting when running mongo are: --dbpath and --wiredTigerCacheSizeGb https://docs.mongodb.com/manual/reference/program/mongod/#wiredtiger-options (ok, there are certainly much more interesting options, but those are really crucial, when running mongo)
- before firing the server you need to create mongo indexes, in general they are currently in catapult-server repo: https://github.com/nemtech/catapult-server/tree/main/scripts/mongo - if you miss this step, db will bring your machine to knees
- broker is generally part of catapult-server repo, you  run it the same as server, passing dir containing resources dir, what is important is config-extensions-broker.properties - I'll get back to this
- server - you run it normally (settings are important, more below)
- rest layer, probably @Steven Liu or @Fernando Boucquez will help you with that, this one should be installable via npm, and you'll need rest.json file with settings (you need to point it at all 3 things - mongo, server - port 7900, broker - port 7902)

---

server settings for **api** node:
- config-extensions-server : addressextraction, mongo, zeromq set to false, config-extensions-broker and config-extensions-recovery should have it set to true
- config-extensions-server : partialtransaction, and filespooling (pushing data to broker) set to true
- config-extensions-server  : eventsource, harvesting, peer, syncsource - all of them are typical for peer node - all should be set to false on api node
- config-node roles should contain Api, enableAutoSyncCleanup set to false
- config-database need to have databaseUri set propertly ofc, btw, this uri can contain user and pass, if mongodb has them set - it would be good if we have any doc discussing it (https://docs.mongodb.com/manual/reference/connection-string/)
- 
server settings for **dual** node (generally as for api node, I'm only listing things that needs to be overridden):
- eventsource, harvesting, syncsource - enabled in config-extensions-server
- roles set to Api, Peer
- I'm guesing on dual node, you usually want to harvest, so enableAutoHarvesting in config-harvesting set to true

on top of that there is **voting** which applies to dual and peer nodes:
- roles need to have , Voting added (so either Peer, Voting or Api, Peer, Voting
- votingKeysDirectory needs to point at a dir containing voting key file (the one generated using votingkey tool) - this is a file with epoch-bound keys used for voting
- enableVoting needs to be set in config-finalization.properties

### PR DESCRIPTION
**Pull request by segfaultxavi**
Fixes #573

### CODE REVIEW

File: source/guides/network/running-a-symbol-node-manually.rst [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+1. **Install** `MongoDB <http://mongodb.com>`__ (at least version 4.4).
+
+   It is recommended to follow the `program's own installation instructions <https://docs.mongodb.com/manual/administration/install-community/>`__, and NOT install from your distro's packages.
+
+2. Create a directory to store the data files and **run the database**:
+
+   .. code-block:: bash
+
+      mkdir dbfiles
+      mongod --dbpath=dbfiles --wiredTigerCacheSizeGb 2 --bind_ip 127.0.0.1
```
Comment (mm-s) [inline source/guides/network/running-a-symbol-node-manually.rst:65]: One of the options is not recognized in my system:
```bash
marc-os@roston:~/mm-s/main/catapult-server/_build/node_7900$ mongod --dbpath=dbfiles --wiredTigerCacheSizeGb 2 --bind_ip 127.0.0.1
Error parsing command line: unrecognised option '--wiredTigerCacheSizeGb'
try 'mongod --help' for more information

marc-os@roston:~/mm-s/main/catapult-server/_build/node_7900$ mongod --version
db version v4.4.4
Build Info: {
    "version": "4.4.4",
    "gitVersion": "8db30a63db1a9d84bdcad0c83369623f708e0397",
    "openSSLVersion": "OpenSSL 1.1.1j  16 Feb 2021",
    "modules": [],
    "allocator": "tcmalloc",
    "environment": {
        "distmod": "debian10",
        "distarch": "x86_64",
        "target_arch": "x86_64"
    }
}
```

Comment (mm-s) [reply source/guides/network/running-a-symbol-node-manually.rst:65]: the offendign option is `--wiredTigerCacheSizeGb 2`. Without it, the process starts correctly (apparently).

File: source/guides/network/running-a-symbol-node-manually.rst [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+   Keep ``mongod`` running for as long as your node runs.
+
+3. **Build indices** to optimize database accesses.
+
+   The ``catapult-server`` repository you cloned to build the server in the first step contains `a folder with mongo scripts <https://github.com/nemtech/catapult-server/blob/main/scripts/mongo>`__. Run this one:
+
+   .. code-block:: bash
+
+      mongo 127.0.0.1/catapult < ../../scripts/mongo/mongoDbPrepare.js
```
Comment (mm-s) [inline source/guides/network/running-a-symbol-node-manually.rst:75]: if the current directory is _build, then the command should be:
`mongo 127.0.0.1/catapult < ../scripts/mongo/mongoDbPrepare.js`
`../../` --> `../`

File: source/guides/network/running-a-symbol-node-manually.rst [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+      databaseUri = mongodb://127.0.0.1:27017
+      databaseName = catapult
+
+5. **Run the broker**:
+
+   Move into the ``bin`` directory and run:
+
+   .. code-block:: bash
+
+      ./catapult.broker ..
```
Comment (mm-s) [inline source/guides/network/running-a-symbol-node-manually.rst:97]: I have a crash here
```
marc-os@roston:~/mm-s/main/catapult-server/_build/bin$ ./catapult.broker ..
Copyright (c) Jaguar0625, gimre, BloodyRookie, Tech Bureau, Corp.
catapult version: 0.10.0.8 ac70cf8e [devnet]
loading resources from "../resources"
loading configuration from "../resources/config-inflation.properties"
loading configuration from "../resources/config-extensions-broker.properties"
loading configuration from "../resources/config-user.properties"
loading configuration from "../resources/config-logging-broker.properties"
loading configuration from "../resources/config-node.properties"
loading configuration from "../resources/config-network.properties"
2021-03-05 21:09:09.475202 0x00007f875c279bc0: <warning> (process::Platform.cpp@39) 
	Current user umask settings are too wide '22'.
	It's strongly suggested to use umask value '077' when starting symbol software. 
2021-03-05 21:09:09.527056 0x00007f875c279bc0: <important> (process::ProcessMain.cpp@90) booting process with keys:
 -   CA EB6003935A7422EAEE04DDD4D7798E076DF672FDA6A3E37A234012C153969230
 - NODE DB5FC52F0F0AF9D8FAF6BF37D4B9FF7CCCA389A59AA34609796576FBF09D255D 
2021-03-05 21:09:09.528391 0x00007f875c279bc0: <info> (thread::IoThreadPool.cpp@105) spawned 8 workers (broker) 
2021-03-05 21:09:09.528717 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libextension.addressextraction.so 
2021-03-05 21:09:09.581109 0x00007f875c279bc0: <info> (extensions::ProcessBootstrapper.cpp@121) registering dynamic extension extension.addressextraction 
2021-03-05 21:09:09.581368 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libextension.mongo.so 
2021-03-05 21:09:09.935788 0x00007f875c279bc0: <info> (extensions::ProcessBootstrapper.cpp@121) registering dynamic extension extension.mongo 
loading configuration from "../resources/config-database.properties"
2021-03-05 21:09:09.973814 0x00007f875c279bc0: <info> (thread::IoThreadPool.cpp@105) spawned 8 workers (bulk writer) 
2021-03-05 21:09:09.985325 0x00007f875c279bc0: <info> (src::MongoPluginLoader.cpp@56) registering dynamic mongo plugin catapult.mongo.plugins.restrictionmosaic 
2021-03-05 21:09:09.985542 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libcatapult.mongo.plugins.restrictionmosaic.so 
2021-03-05 21:09:09.988909 0x00007f875c279bc0: <info> (src::MongoPluginLoader.cpp@56) registering dynamic mongo plugin catapult.mongo.plugins.restrictionaccount 
2021-03-05 21:09:09.989056 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libcatapult.mongo.plugins.restrictionaccount.so 
2021-03-05 21:09:09.990440 0x00007f875c279bc0: <info> (src::MongoPluginLoader.cpp@56) registering dynamic mongo plugin catapult.mongo.plugins.multisig 
2021-03-05 21:09:09.990466 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libcatapult.mongo.plugins.multisig.so 
2021-03-05 21:09:09.991047 0x00007f875c279bc0: <info> (src::MongoPluginLoader.cpp@56) registering dynamic mongo plugin catapult.mongo.plugins.mosaic 
2021-03-05 21:09:09.991075 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libcatapult.mongo.plugins.mosaic.so 
2021-03-05 21:09:09.991640 0x00007f875c279bc0: <info> (src::MongoPluginLoader.cpp@56) registering dynamic mongo plugin catapult.mongo.plugins.metadata 
2021-03-05 21:09:09.991669 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libcatapult.mongo.plugins.metadata.so 
2021-03-05 21:09:09.992286 0x00007f875c279bc0: <info> (src::MongoPluginLoader.cpp@56) registering dynamic mongo plugin catapult.mongo.plugins.locksecret 
2021-03-05 21:09:09.992312 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libcatapult.mongo.plugins.locksecret.so 
2021-03-05 21:09:09.992908 0x00007f875c279bc0: <info> (src::MongoPluginLoader.cpp@56) registering dynamic mongo plugin catapult.mongo.plugins.lockhash 
2021-03-05 21:09:09.992933 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libcatapult.mongo.plugins.lockhash.so 
2021-03-05 21:09:09.993516 0x00007f875c279bc0: <info> (src::MongoPluginLoader.cpp@56) registering dynamic mongo plugin catapult.mongo.plugins.aggregate 
2021-03-05 21:09:09.993543 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libcatapult.mongo.plugins.aggregate.so 
2021-03-05 21:09:09.993824 0x00007f875c279bc0: <info> (src::MongoPluginLoader.cpp@56) registering dynamic mongo plugin catapult.mongo.plugins.transfer 
2021-03-05 21:09:09.993849 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libcatapult.mongo.plugins.transfer.so 
2021-03-05 21:09:09.994105 0x00007f875c279bc0: <info> (src::MongoPluginLoader.cpp@56) registering dynamic mongo plugin catapult.mongo.plugins.namespace 
2021-03-05 21:09:09.994128 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libcatapult.mongo.plugins.namespace.so 
2021-03-05 21:09:09.994743 0x00007f875c279bc0: <info> (src::MongoPluginLoader.cpp@56) registering dynamic mongo plugin catapult.mongo.plugins.accountlink 
2021-03-05 21:09:09.994768 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libcatapult.mongo.plugins.accountlink.so 
2021-03-05 21:09:09.995916 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libextension.zeromq.so 
2021-03-05 21:09:10.108912 0x00007f875c279bc0: <info> (extensions::ProcessBootstrapper.cpp@121) registering dynamic extension extension.zeromq 
loading configuration from "../resources/config-messaging.properties"
2021-03-05 21:09:10.173560 0x00007f875c279bc0: <info> (thread::IoThreadPool.cpp@105) spawned 1 workers (ZeroMqEntityPublisher) 
2021-03-05 21:09:10.173808 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libextension.hashcache.so 
2021-03-05 21:09:10.200215 0x00007f875c279bc0: <info> (extensions::ProcessBootstrapper.cpp@121) registering dynamic extension extension.hashcache 
2021-03-05 21:09:10.200519 0x00007f875c279bc0: <info> (broker::Broker.cpp@61) registering system plugins 
2021-03-05 21:09:10.200640 0x00007f875c279bc0: <info> (plugins::PluginLoader.cpp@48) registering dynamic plugin catapult.plugins.coresystem 
2021-03-05 21:09:10.200737 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libcatapult.plugins.coresystem.so 
2021-03-05 21:09:10.207257 0x00007f875c279bc0: <info> (plugins::PluginLoader.cpp@48) registering dynamic plugin catapult.plugins.signature 
2021-03-05 21:09:10.207371 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libcatapult.plugins.signature.so 
2021-03-05 21:09:10.209322 0x00007f875c279bc0: <info> (plugins::PluginLoader.cpp@48) registering dynamic plugin catapult.plugins.hashcache 
2021-03-05 21:09:10.209391 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libcatapult.plugins.hashcache.so 
2021-03-05 21:09:10.211911 0x00007f875c279bc0: <info> (plugins::PluginLoader.cpp@48) registering dynamic plugin catapult.plugins.transfer 
2021-03-05 21:09:10.211986 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libcatapult.plugins.transfer.so 
2021-03-05 21:09:10.214080 0x00007f875c279bc0: <info> (plugins::PluginLoader.cpp@48) registering dynamic plugin catapult.plugins.namespace 
2021-03-05 21:09:10.214142 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libcatapult.plugins.namespace.so 
2021-03-05 21:09:10.217431 0x00007f875c279bc0: <info> (plugins::PluginLoader.cpp@48) registering dynamic plugin catapult.plugins.restrictionmosaic 
2021-03-05 21:09:10.217491 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libcatapult.plugins.restrictionmosaic.so 
2021-03-05 21:09:10.220213 0x00007f875c279bc0: <info> (plugins::PluginLoader.cpp@48) registering dynamic plugin catapult.plugins.metadata 
2021-03-05 21:09:10.220263 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libcatapult.plugins.metadata.so 
2021-03-05 21:09:10.222537 0x00007f875c279bc0: <info> (plugins::PluginLoader.cpp@48) registering dynamic plugin catapult.plugins.multisig 
2021-03-05 21:09:10.222583 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libcatapult.plugins.multisig.so 
2021-03-05 21:09:10.224672 0x00007f875c279bc0: <info> (plugins::PluginLoader.cpp@48) registering dynamic plugin catapult.plugins.lockhash 
2021-03-05 21:09:10.224715 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libcatapult.plugins.lockhash.so 
2021-03-05 21:09:10.226840 0x00007f875c279bc0: <info> (plugins::PluginLoader.cpp@48) registering dynamic plugin catapult.plugins.mosaic 
2021-03-05 21:09:10.226882 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libcatapult.plugins.mosaic.so 
2021-03-05 21:09:10.228907 0x00007f875c279bc0: <info> (plugins::PluginLoader.cpp@48) registering dynamic plugin catapult.plugins.locksecret 
2021-03-05 21:09:10.228942 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libcatapult.plugins.locksecret.so 
2021-03-05 21:09:10.230817 0x00007f875c279bc0: <info> (plugins::PluginLoader.cpp@48) registering dynamic plugin catapult.plugins.accountlink 
2021-03-05 21:09:10.230852 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libcatapult.plugins.accountlink.so 
2021-03-05 21:09:10.232102 0x00007f875c279bc0: <info> (plugins::PluginLoader.cpp@48) registering dynamic plugin catapult.plugins.restrictionaccount 
2021-03-05 21:09:10.232136 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libcatapult.plugins.restrictionaccount.so 
2021-03-05 21:09:10.234073 0x00007f875c279bc0: <info> (plugins::PluginLoader.cpp@48) registering dynamic plugin catapult.plugins.aggregate 
2021-03-05 21:09:10.234104 0x00007f875c279bc0: <info> (plugins::PluginModule.cpp@83) loading plugin from ./libcatapult.plugins.aggregate.so 
2021-03-05 21:09:10.234974 0x00007f875c279bc0: <info> (utils::StackLogger.h@37) pushing scope 'booting broker' 
2021-03-05 21:09:10.235214 0x00007f875c279bc0: <info> (utils::StackLogger.h@43) popping scope 'booting broker' (0ms) 
2021-03-05 21:09:10.235277 0x00007f875c279bc0: <info> (process::Signals.cpp@45) waiting for termination signal 
2021-03-05 21:09:10.336176 0x00007f875926d700: <error> (io::FileQueue.cpp@133) Throwing exception: reading from file queue failed due to missing message file ("../data/spool/state_change/0000000000000000.dat") 
2021-03-05 21:09:10.340484 0x00007f875926d700: <fatal> (process::ProcessMain.cpp@75) 
thread: 5 (broker) work
unhandled exception while running local node!
../src/catapult/io/FileQueue.cpp(133): Throw in function bool catapult::io::FileQueueReader::process(catapult::consumer<const std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&>&)
Dynamic exception type: boost::wrapexcept<catapult::catapult_error<std::runtime_error> >
std::exception::what: reading from file queue failed due to missing message file
[catapult::ErrorParam1*] = "../data/spool/state_change/0000000000000000.dat"
 
```

Comment (mm-s) [reply source/guides/network/running-a-symbol-node-manually.rst:97]:
```bash
total 24
drwx------ 2 marc-os marc-os 4096 Mar  5 21:16 .
drwx------ 9 marc-os marc-os 4096 Mar  5 21:09 ..
-rw------- 1 marc-os marc-os    8 Mar  5 21:09 index_broker_r.dat
-rw------- 1 marc-os marc-os    8 Mar  5 21:16 index.dat
-rw------- 1 marc-os marc-os    8 Mar  5 21:16 index_server.dat
-rw------- 1 marc-os marc-os    8 Mar  5 21:16 index_server_r.dat

```

Comment (mm-s) [reply source/guides/network/running-a-symbol-node-manually.rst:97]: processes catapult-server and mongodb are running fine at any moment before and after the broker crash

File: source/guides/network/running-a-symbol-node-manually.rst [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+==============================
+
+6. **Clone** the ``catapult-rest`` repository and **install its dependencies**:
+
+   .. code-block:: bash
+
+      git clone https://github.com/nemtech/catapult-rest.git
+      cd catapult-rest
+      npm install -g yarn
+      ./yarn-setup.sh
```
Comment (mm-s) [inline source/guides/network/running-a-symbol-node-manually.rst:111]: typo. the correct form is `./yarn_setup.sh`

File: source/guides/network/running-a-symbol-node-manually.rst [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+
+      {
+         "host": "127.0.0.1",
+         "port": 7900,
+         "timeout": 1000,
+         "tlsClientCertificatePath": "~/catapult-server/_build/certificate/node.crt.pem",
+         "tlsClientKeyPath": "~/catapult-server/_build/certificate/node.key.pem",
+         "tlsCaCertificatePath": "~/catapult-server/_build/certificate/ca.cert.pem",
+         "networkPropertyFilePath": "~/catapult-server/_build/resources/config-network.properties",
+         "nodePropertyFilePath": "~/catapult-server/_build/resources/config-node.properties"
```
Comment (mm-s) [inline source/guides/network/running-a-symbol-node-manually.rst:149]: the use of "~/" makes the gateway exit with
```
error: uncaughtException: ENOENT: no such file or directory, open '~/mm-s/main/catapult-server/_build/certificate/node.crt.pem'
```
however, if the path is fully expanded as in
```
   "tlsClientCertificatePath": "/home/marc-os/mm-s/main/catapult-server/_build/certificate/node.crt.pem",
    "tlsClientKeyPath": "/home/marc-os/mm-s/main/catapult-server/_build/certificate/node.key.pem",
    "tlsCaCertificatePath": "/home/marc-os/mm-s/main/catapult-server/_build/certificate/ca.cert.pem",
    "networkPropertyFilePath": "/home/marc-os/mm-s/main/catapult-server/_build/resources/config-network.properties",
    "nodePropertyFilePath": "/home/marc-os/mm-s/main/catapult-server/_build/resources/config-node.properties"

```
It does work well

File: source/guides/network/running-a-symbol-node-manually.rst [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+
+  .. code-block:: json
+
+     {
+        "status":{
+           "apiNode":"up",
+           "db":"up"
+        }
+     }
+
```
Comment (mm-s) [inline source/guides/network/running-a-symbol-node-manually.rst:177]: perhaps outdated?
I can't see any status info in the JSON I get from port 3000:

```
version | 655368
-- | --
publicKey | "EB6003935A7422EAEE04DDD4D7798E076DF672FDA6A3E37A234012C153969230"
networkGenerationHashSeed | "4DF76A205008026C776CB6AFD6D3393F04CD458E0DD2DDFDDDF31A402072B2D6"
roles | 65
port | 7900
networkIdentifier | 168
host | "192.168.0.38"
friendlyName | "nodish1"
nodePublicKey | "DB5FC52F0F0AF9D8FAF6BF37D4B9FF7CCCA389A59AA34609796576FBF09D255D"

```
?

File: source/guides/network/running-a-symbol-node-manually.rst (hunk 1 above)
Comment (segfaultxavi) [reply source/guides/network/running-a-symbol-node-manually.rst:65]: Yeah, the right option is `--wiredTigerCacheSizeGB`

File: source/guides/network/running-a-symbol-node-manually.rst (hunk 2 above)
Comment (segfaultxavi) [reply source/guides/network/running-a-symbol-node-manually.rst:75]: 👍

File: source/guides/network/running-a-symbol-node-manually.rst (hunk 3 above)
Comment (segfaultxavi) [reply source/guides/network/running-a-symbol-node-manually.rst:97]: I don't know what to say here... did restarting the broker fox the issue? @gimer @Jaguar0625

File: source/guides/network/running-a-symbol-node-manually.rst (hunk 4 above)
Comment (segfaultxavi) [reply source/guides/network/running-a-symbol-node-manually.rst:111]: 👍

File: source/guides/network/running-a-symbol-node-manually.rst (hunk 5 above)
Comment (segfaultxavi) [reply source/guides/network/running-a-symbol-node-manually.rst:149]: Good to know!

File: source/guides/network/running-a-symbol-node-manually.rst (hunk 6 above)
Comment (segfaultxavi) [reply source/guides/network/running-a-symbol-node-manually.rst:177]: That does not look like JSON at all, did you really get it from http://localhost:3000/node/info ?

File: source/guides/network/running-a-symbol-node-manually.rst [hunk 7]
```diff
@@ -0,0 +1,409 @@
...
+           "finalizationEpoch": 224,
+           "finalizationPoint": 44,
+           "height": "160560",
+           "hash": "52D3B01920C695B9194FABD869804E4D2A18D9B5509E47B2C70B0E6C3E275E33"
+        }
+     }
+
+  The ``height`` field should match the actual height of the blockchain, but, upon first starting the node, it might take a while to synchronize.
+
+Finally, go to the `Symbol Explorer page <http://explorer.symbolblockchain.io/nodes>`__ and check that your node appears in the list (It refreshes very 30 seconds).
```
Comment (OlegMakarenko) [inline source/guides/network/running-a-symbol-node-manually.rst:409]:
```suggestion
Finally, go to the `Symbol Explorer page <http://explorer.symbolblockchain.io/nodes>`__ and check that your node appears in the list (It refreshes every 30 seconds).
```

**Review by OlegMakarenko — CHANGES_REQUESTED** [review]

File: source/guides/network/running-a-symbol-node-manually.rst (hunk 7 above)
Comment (segfaultxavi) [reply source/guides/network/running-a-symbol-node-manually.rst:409]: Oops!

**Pull request merged**